autosuspend.py

```diff
@@ -272,7 +272,7 @@ def check(self):
                 if line.startswith('----'):
                     start_seen = True
 
-        if connections:
+        if len(connections) > 1:
```

Comment:
Why is this necessary?

Reply:
this Smb check is (on my machine (samba 4.7.0)) always true, because the output of 'smbstatus -b'ends with /n. Looks like Split finds minimum one line after the '----' ?

Reply:
@das-nervt that is weird. Could you please provide the raw output of `smbstatus -b` in case no one is connected

Reply:
Ok, I see the issue. Weird that I did not notice this on my own. I will address this separately. Could you please remove this commit from the PR as this is unrelated to the `psutils` issue.